content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/020-use-custom-model-and-field-names.mdx

```diff
@@ -173,7 +173,7 @@ You can "rename" fields and models that are used in Prisma Client by mapping the
 
 _After_ you introspected your database with `prisma db pull`, you can manually adjust the resulting Prisma schema as follows:
 
-```prisma
+```prisma highlight=1-2;edit|4;edit|5-6;add|9;edit|12-13;edit|16;add|19-20;edit|25-26;edit|27-28;add|31-32;edit|35;edit|36-27;add|40-41;edit|44-45;edit|46-47;add
```

Comment:
The code snippet is too chaotic, and hard to follow the changes. I would suggest either:
- Using tabs — one tab containing the updates and the other one for the final schema (no highlights)
- A combo of a comment (right above the relation field) and highlight for the section with the `@@map("")` attribute function
- Using a simpler (our [standard](https://docs-git-jharrell-issue3815-prisma.vercel.app/about/prisma-docs/style-guide/schema-models)) schema with the Post & User models


https://docs-git-jharrell-issue3815-prisma.vercel.app/concepts/components/prisma-client/working-with-prismaclient/use-custom-model-and-field-names#using-map-and-map-to-rename-fields-and-models-in-the-prisma-client-api

<img width="481" alt="CleanShot 2023-07-25 at 09 30 21@2x" src="https://github.com/prisma/docs/assets/33921841/de4f9529-2f25-469e-976d-a4d02da063c3">